e2e-tests/js_scripts/0008-update-runtime-multisig-scheduler.js

```diff
@@ -0,0 +1,153 @@
+    const alice = keyring.addFromUri('//Alice');
+    const bob = keyring.addFromUri('//Bob');
+    const charlie = keyring.addFromUri('//Charlie');
+    const ACME = 1_000_000_000_000_000_000n;
+
+    /*****************************************************************************************************
+     *************************************** CREATE MULTISIG ACCOUNT *************************************
+     *****************************************************************************************************/
+    const threshold = 2;
+    const multisigAddress = util.createKeyMulti([alice.address, bob.address, charlie.address], threshold);
+    const SS58Prefix = 42;
+    const Ss58MultiAddress = util.encodeAddress(multisigAddress, SS58Prefix);
+    console.log(`multisigAddress ${Ss58MultiAddress}`);
+
+
+    /*****************************************************************************************************
+     *************************************** SET MULTISIG AS SUDO ACCOUNT ********************************
+     *****************************************************************************************************/
+
+    const newSudoKey = api.tx.sudo.setKey(Ss58MultiAddress);
+    await submitExtrinsic(newSudoKey, alice, BlockUntil.Finalized, undefined);
+
+    const newSudoKeyOption = await api.query.sudo.key();
+    if (newSudoKeyOption.isSome && newSudoKeyOption.unwrap().toString() !== Ss58MultiAddress) {
+        return ReturnCode.ErrWrongSudoAccount;
+    }
+
+    /*****************************************************************************************************
+     *************************************** GET CURRENT RUNTIME VERSION******* **************************
+     *****************************************************************************************************/
+
+    const currentRuntimeVersion = (await api.rpc.state.getRuntimeVersion()).specVersion.toNumber();
+
+    /*****************************************************************************************************
+     *************************************** SCHEDULE RUNTIME UPGRADE*************************************
+     *****************************************************************************************************/
+
+    // Retrieve the runtime to upgrade
+    const sudoAccount = await api.query.sudo.key()
+    const code = fs.readFileSync('./new-runtime/nh_runtime.wasm').toString('hex');
+    const updateRuntimeCall = api.tx.system && api.tx.system.setCode
+        ? api.tx.system.setCode(`0x${code}`) // For newer versions of Substrate
+        : api.tx.consensus.setCode(`0x${code}`); // For previous versions
+    const sudoUpdateRuntimeCall = api.tx.sudo.sudo(updateRuntimeCall)
+
+    console.log(`Upgrading from ${sudoAccount}, ${code.length / 2} bytes`);
+
+    const { block } = await api.rpc.chain.getBlock();
+    const blockNumber = block.header.number.toNumber();
+
+    const when = blockNumber + 10;
+    const priority = 100;
```

Comment:
```suggestion
    const when = blockNumber + 2;
```
Maybe is better... 48 seconds faster :smile:

Reply:
Neremind.... sorry

Reply:
Ok I did some tests use 
```suggestion
    const when = blockNumber + 5;
```

should be safe if we don't wait for finalization in all transactions